common-files/archetype-post-generate.groovy

```diff
@@ -0,0 +1,26 @@
+import java.nio.file.Path
+import java.nio.file.Paths
+
+Path projectPath = Paths.get(request.outputDirectory, request.artifactId)
+Properties properties = request.properties
+
+if (properties.get("hostOnJenkinsGitHub") == "false") {
+    println "Not hosting on Jenkins Github organisation, removing files specific to jenkinsci"
+
+    def filesToRemove = [
+            'Jenkinsfile',
+    ]
+    def directoriesToRemove = [
+            '.github',
+            '.dependabot',
+            '.mvn'
```

Comment:
If you delete `.mvn/` then the incrementals config in the POM is going to be just a confusing and useless complication. (And possibly broken, in the case of `scmTag` anyway.)